src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1737,11 +1743,21 @@ def run(self):
       # is using command override for task execution.
       failure_wait_interval = environment.get_value('FAIL_WAIT')
       time.sleep(failure_wait_interval)
-      return uworker_msg_pb2.Output(  # pylint: disable=no-member
+      return uworker_msg_pb2.Output(  # pylint: disable=no-memberp
           error_type=uworker_msg_pb2.ErrorType.FUZZ_NO_FUZZER)  # pylint: disable=no-member
 
     self.testcase_directory = environment.get_value('FUZZ_INPUTS')
+    return None
 
+  def run(self):
+    """Run the fuzzing session."""
+    start_time = time.time()
+    result = self._run_common()
+    if result:
+      return result
```

Comment:
Probably worth moving the metric from #4336 to a decorator as you pointed out, we are losing instrumentation in this early return.